float64[3] acc
float64[3] gyr
float64[3] accTruth
float64[3] gyrTruth
float64 tStamp
